packages/serverless/src/gcpfunction.ts

```diff
@@ -0,0 +1,115 @@
+import { HttpFunction } from '@google-cloud/functions-framework/build/src/functions';
+import { captureException, flush, getCurrentHub, Scope, SDK_VERSION, startTransaction, withScope } from '@sentry/node';
+import { Transaction } from '@sentry/types';
+import { addExceptionMechanism, logger, stripUrlQueryAndFragment } from '@sentry/utils';
+import { Domain } from 'domain';
+import { Request, Response } from 'express'; // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
Is this import magically available on GCP?

Reply:
It's a type-only import too. But here I think I could just remove this import at all and just retrieve these types from `Parameters<HttpFunction>`, e.g.:

```typescript
type Request = Parameters<HttpFunction>[0];
type Response = Parameters<HttpFunction>[1];
```